packages/playground/src/calculator/pricing_calculator.vue

```diff
@@ -153,28 +153,43 @@
           <VRow class="text-center text-body-1 text-black" v-else-if="valid">
             <VCol cols="6">
               <div
-                class="rounded pa-4 h-100 d-flex justify-center align-center"
-                :style="{ background: computePackageColor(priceTask.data?.dedicatedPackage.package) }"
+                class="rounded pa-4 discount"
+                :style="{
+                  background: computePackageColor(priceTask.data?.dedicatedPackage.package),
+                }"
               >
                 <p>
-                  Cost of reserving a <strong v-text="'Dedicated Node'" /> of the same specifications<br />
+                  Cost of reserving a <strong v-text="'Dedicated Node '" />
                   <strong v-text="'Loading...'" v-if="priceTask.loading" />
-                  <strong v-else v-text="dedicatedPriceUSD + ' USD/month, ' + dedicatedPriceTFT + ' TFT/month'" />. A
-                  user can reserve an entire node then use it exclusively to deploy solutions
+                  <strong v-else v-text="dedicatedPriceUSD + ' USD/month, ' + dedicatedPriceTFT + ' TFT/month.'" />
                 </p>
+                <section class="card" v-if="priceTask.data?.dedicatedPackage.package !== 'gold'">
+                  <p class="card-info">
+                    <b>Too expensive?</b> can upgrade to <b>Gold package</b> to get discount up to 60% when you fund
+                    your wallet with <b>{{ Math.ceil(114.473 * 18) }}</b> TFT
```

Comment:
Are you sure about this?
i see in the pricing docs the value to get `40%` discount is `120,000`
https://manual.grid.tf/knowledge_base/cloud/pricing/staking_discount_levels.html#example-for-40-discount

Reply:
yes, it's about ```60%``` discount
https://www.manual.grid.tf/knowledge_base/cloud/pricing/pricing.html#staking-discount